plexapi/server.py

```diff
@@ -363,6 +364,30 @@ def playlist(self, title):
         """
         return self.fetchItem('/playlists', title=title)
 
+    def optimizedItems(self):
+        """ Returns list of all :class:`~plexapi.media.Optimized` objects connected to server. """
+        items = []
+
+        backgroundProcessing = self.query('/playlists?type=42')
+        for elem in backgroundProcessing:
+            key = elem.attrib.get('key')
+            for elem in self.query(key):
+                items.append(Optimized(server=self, data=elem))
+
+        return items
+
+    def conversions(self):
+        """ Returns list of all :class:`~plexapi.media.Conversion` objects connected to server. """
+        items = []
+
+        backgroundProcessing = self.query('/playQueues?type=42')
+        for elem in backgroundProcessing:
+            key = elem.attrib.get('id')
+            for elem in self.query('/playQueues/%s' % key):
```

Comment:
fetchItem

Reply:
If we can assume that `/playQueues/1` is the default queue for conversions then
```python
return self.fetchItems('/playQueues/1')
```

or
```python
backgroundProcessing = self.query('/playQueues?type=42')
key = backgroundProcessing[0].attrib.get('id')
return self.fetchItems('/playQueues/%s' % key)
```
Either would return a partial `Video` object and not the `Conversion` object that I've created. So some information is gained and some is lost (video vs conversion object). The conversion object is true to the xml that is retrieved. Unless there is another way to force fetchItems to use the Conversion object, to which I'm struggling to find.

Reply:
The playqueue needs a rewamp, we are missing move, and add. I think i need  this for bw_plex so im gonna send a pr for that later. As for the fetchItems i think its a cls parameter that you can use.